README.md

```diff
@@ -21,6 +21,7 @@ addon | version | summary
 --- | --- | ---
 [base_location](base_location/) | 9.0.1.0.0 | Enhanced zip/npa management system
 [partner_contact_nationality](partner_contact_nationality/) | 9.0.1.0.0 | Add nationality field to contacts
+[partner_firstname](partner_firstname/) | 9.0.2.0.0 | Split first name and last name for non company partners
 
```

Comment:
hi @kkarolis Thanks for your contribution to the OCA. 
Please follow the guidelines from:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md

There should be no md file (for repo), only a README.rst, following: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst